web/themes/custom/usagov/templates/node--1--full.html.twig

```diff
@@ -11,7 +11,9 @@
 		{% include directory ~ '/templates/includes/homepage-all-topics-jump.html.twig' %}
 	</div>
 
-	{% include directory ~ '/templates/includes/homepage-life-events.html.twig' %}
```

Comment:
We will want an if/else clause here based on the configuration switch @omerida's going to implement.

Reply:
Are we going to push these changes early and just "hide" them with the switch?

Reply:
That's the idea!